src/api/server/mod.rs

```diff
@@ -217,7 +218,7 @@ impl BriaService for Bria {
                 destination.try_into()?,
                 Satoshis::from(satoshis),
                 None,
-                None,
+                metadata.map(|metadata| serde_json::to_value(metadata).unwrap()),
```

Comment:
Need to confirm how we handle errors currently

Reply:
never - EVER - ever call `unwrap` in production code. Even if you are 100% sure it can never error you should use `expect` but in 99.9% of cases just use `?` instead.

Reply:
Yup I wasn't sure as yet how to handle those errors. I think I figure it out broadly here, but not sure if I used the right Status code and an appropriate message https://github.com/GaloyMoney/bria/pull/121/commits/d2bf6aaa21a9257a703737eb6a28a087fc280292